### ISSUE TITLE
conversion from numpy array to tensor

### ISSUE THREAD

**Issue by bndav** [biomed-AI/GraphEC#3]
Hello,

To successfully use the program, I had to add the following line:
prottrans_feat = torch.from_numpy(prottrans_feat)

Before the line:
pre_computed_node_feat = torch.cat([prottrans_feat, dssp_feat], dim=-1)

In both files:
EC_number/data.py
Active_sites/data.py
